CHANGELOG.md

```diff
@@ -2,10 +2,19 @@
 
 <!-- prettier-ignore-start -->
 > [!IMPORTANT]
-> If you are upgrading to the `1.x` versions of the SDK from `0.x`, make sure you follow our
+> If you are upgrading to the `2.x` versions of the SDK from `1.x` or lower, make sure you follow our
 > [migration guide](https://docs.sentry.io/platforms/javascript/guides/capacitor/migration/) first.
 <!-- prettier-ignore-end -->
 
+## Unreleased
+
+### Dependencies
+
+- Bump JavaScript SDKs from v8.42.1 to v9.0.0 ([#840](https://github.com/getsentry/sentry-capacitor/pull/840))
+
+  - [changelog](https://github.com/getsentry/sentry-javascript/blob/9.0.0/CHANGELOG.md#900)
+  - [diff](https://github.com/getsentry/sentry-javascript/compare/8.42.0...9.0.0)
+
```

Comment:
We could also add the migration guide to the list.

````suggestion
  - [migration guide](https://docs.sentry.io/platforms/javascript/migration/v8-to-v9/)

````

Reply:
Actually, we can update the changelog when https://github.com/getsentry/sentry-docs/pull/13022 is merged.